website/frontend/src/utils/tokenizer.ts

```diff
@@ -0,0 +1,83 @@
+// Simple LLama2 tokenizer implementation
+let tokenizer: { vocab_reverse: Record<number, string> } | null = null
+
+export async function loadTokenizer() {
+	if (tokenizer) return tokenizer
+
+	try {
+		const response = await fetch('/llama2_tokenizer.json')
```

Comment:
Just to know, can we fetch this from Hugging Face URL directly?
If not, when a run uses another model, the text wont make sense
If it is too complex, we can do it in another PR